### PR TITLE
feat(helpers): adds batch insert and quoting helpers

### PR DESCRIPTION
**Pull request by andrewxhill**
based a couple bits of chatter in dev-help and twitter the past few weeks, made a couple enhancements this weekend. 

new batch helper allows for more easily managing a batch insert from an array of value strings. 

the quote method is just removing one more bit of concatenation complexity from the end user now. 

drops a couple unneeded spaces for gas too.

### CODE REVIEW

**Review by joewagner — APPROVED** [review]
Very nice!  This will be useful for a lot of folks

File: test/utils/SQLHelpers.ts [hunk 1]
```diff
@@ -39,7 +38,19 @@ describe("SQLHelpers", function () {
...
+
+  it("Should return a valid INSERT statement from columns and an array of values", async function () {
+    await expect(
+      // This is not a valid name in Tableland but tests string concat.
+      await lib.toBatchInsert("test_101", 1, "id, name, desc", [
+        "1,'hello','information'",
+        "2,'world','information'",
+      ])
+    ).to.equal(
+      "INSERT INTO test_101_31337_1(id, name, desc)VALUES(1,'hello','information'),(2,'world','information')"
```
Comment (sanderpick) [inline test/utils/SQLHelpers.ts:53]: possible to remove the spaces between the column names?

**Review by sanderpick — DISMISSED** [review]
nice! approved but added one question

File: test/utils/SQLHelpers.ts (hunk 1 above)
Comment (andrewxhill) [reply test/utils/SQLHelpers.ts:53]: totally, good catch. will update

Comment (andrewxhill) [reply test/utils/SQLHelpers.ts]: all done

**Review by sanderpick — APPROVED** [review]

**Pull request merged**